src/templates/docx/LDCETemplate.ts

```diff
@@ -74,7 +74,9 @@ export function LDCETemplate(passport: Passport): Document {
         fields: [
           {
             name: "Is the applicant’s address the same as the site address?",
-            value: get("applicant.sameAddress.form"),
+            value:
+              get("applicant.sameAddress.form") ||
+              get("applicant.address.sameAsSiteAddress"),
```

Comment:
This is written into content quite strangely, and was failing to pick up "No" before https://editor.planx.uk/opensystemslab/about-the-applicant
![Screenshot from 2023-11-02 15-02-52](https://github.com/theopensystemslab/planx-core/assets/5132349/0ab43814-c5a0-4b3e-9837-28346528d0c2)